File: hackerrank/medium/0002-extra-long-factorials/js/solution.mjs
// solution.mjs

export function extraLongFactorials(n) {
  let res = 1n;
  for (let i = 2n; i <= BigInt(n); i++) res *= i;
  console.log(res.toString());
}

// --- demo run ---
extraLongFactorials(10); // 3628800
extraLongFactorials(25); // 15511210043330985984000000

File: hackerrank/medium/0002-extra-long-factorials/python/solution.py
## solution.py

def extra_long_factorials(n: int) -> None:
    res = 1
    for i in range(2, n + 1):
        res *= i
    print(res)


# --- demo run ---
if __name__ == "__main__":
    extra_long_factorials(10)  # 3628800
    extra_long_factorials(25)  # 15511210043330985984000000